packages/react-instantsearch-core/src/core/__tests__/createInstantSearchManager.js

```diff
@@ -68,6 +68,12 @@ describe('createInstantSearchManager', () => {
         _cache: true, // cache is not enabled by default inside Node
       });
 
+      // Skip this test with Algoliasearch API Client >= v4
```

Comment:
I feel like we should adapt the tests for client v4 usage, here we don't test anything related to SSR working with RIS + Client v4.

Reply:
Yeah I agree. We should write an integration test here I believe.